Bluewire.Tools.Runner/Shared/Build.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Bluewire.Common.GitWrapper.Model;
+using Bluewire.Conventions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bluewire.Tools.Runner.Shared
+{
+    public struct Build
+    {
+        /// <summary>
+        /// Commit hash
+        /// </summary>
+        public Ref Commit { get; set; }
+
+        /// <summary>
+        /// Semantic version representing the location of the Commit in the git tree
+        /// </summary>
+        public SemanticVersion SemanticVersion { get; set; }
+
+        public override string ToString()
+        {
+            return string.Format("{0} {1}", Commit.ToString(), SemanticVersion.ToString());
+        }
+
+        public static Build[] DeduplicateAndPrioritiseResult(Build[] builds)

```

Comment:
Yeah I swayed between this being a class and struct many times - it's borderline IMO but I've tried to keep it as a struct to follow the conventions used elsewhere in this project and should have moved this static function into a new class once I settled on it being a struct. It's now in `BuildUtils.cs`